src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+)
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The revision number of the image release"`
+}
+
+var opts struct {
+	// TODO: Leave `release` here for now. Should be moved into a `cmd_release.go` in phase 2.
+	CmdRelease       CmdRelease `command:"release" description:"Release (re-tag) the image into the container registries"`
+	SkipConfirmation bool       `short:"y" description:"Skip the confirmation to upload/release an image"`
+}
+
+var parser = flags.NewParser(&opts, flags.Default)
+
+func CliMain() {
+	if len(os.Args) == 1 || os.Args[1] == "help" {
+		parser.WriteHelp(os.Stdout)
```

Comment:
We're missing a general description of the `oci-factory` client in the help text.

Reply:
Fixed.